tsconfig.json

```diff
@@ -1,12 +1,19 @@
 {
   "compilerOptions": {
-    "target": "es5",
-    "module": "commonjs",
     "sourceMap": true,
     "strict": true,
-    "noImplicitAny": false,
     "experimentalDecorators": true,
     "strictNullChecks": true,
-    "esModuleInterop": true
-  }
+    "esModuleInterop": true,
+    "noImplicitAny": true,
+    "module": "es6",
+    "target": "es5",
+    "jsx": "react",
+    //"allowJs": true,
+    "declaration": true,
+    "declarationDir": "./types"
```

Comment:
@paolobueno I will check if we can generate types in the way users could use them in their apps. I think that types will be much better than exposing typescript code (unstable etc.)